Fix override logging lines

C0rn3j opened the pull request:
Since they're Path now it fails to concat.

msmafra commented:
https://github.com/Taiko2k/Tauon/issues/1437#issuecomment-2660169199

> [@msmafra](https://github.com/msmafra) Not related.
> 
> You can do `unset XDG_MUSIC_DIR` before launch for now, fixed by [#1440](https://github.com/Taiko2k/Tauon/pull/1440)

Actually I commented out all XDG dirs definitions for it to work. Did not test one by one, but commenting out only XDG_MUSIC_DIR did not do the trick.

C0rn3j commented:
XDG_DOWNLOAD_DIR had the same issue, rest should work

On Tue, Feb 18, 2025, 00:23 Marcelo dos Santos Mafra <
***@***.***> wrote:

> #1437 (comment)
> <https://github.com/Taiko2k/Tauon/issues/1437#issuecomment-2660169199>
>
> @msmafra <https://github.com/msmafra> Not related.
>
> You can do unset XDG_MUSIC_DIR before launch for now, fixed by #1440
> <https://github.com/Taiko2k/Tauon/pull/1440>
>
> Actually I commented out all XDG dirs definitions for it to work. Did not
> test one by one, but commenting out only XDG_MUSIC_DIR did not do the trick.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Taiko2k/Tauon/pull/1440#issuecomment-2664220047>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMQXERNML3ZDBR3A6PSJ3T2QJVQBAVCNFSM6AAAAABXFLR7M2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRUGIZDAMBUG4>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
> [image: msmafra]*msmafra* left a comment (Taiko2k/Tauon#1440)
> <https://github.com/Taiko2k/Tauon/pull/1440#issuecomment-2664220047>
>
> #1437 (comment)
> <https://github.com/Taiko2k/Tauon/issues/1437#issuecomment-2660169199>
>
> @msmafra <https://github.com/msmafra> Not related.
>
> You can do unset XDG_MUSIC_DIR before launch for now, fixed by #1440
> <https://github.com/Taiko2k/Tauon/pull/1440>
>
> Actually I commented out all XDG dirs definitions for it to work. Did not
> test one by one, but commenting out only XDG_MUSIC_DIR did not do the trick.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Taiko2k/Tauon/pull/1440#issuecomment-2664220047>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMQXERNML3ZDBR3A6PSJ3T2QJVQBAVCNFSM6AAAAABXFLR7M2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRUGIZDAMBUG4>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>
